### DISCUSSION TITLE
Will the charging station location be changing for the final test for scoring or will it remain in the same location for all the tests?

### DISCUSSION THREAD

**Discussion by DerryLin-Tamu** [SimDynamX/SpaceTeamsPRO#85] [2024 Lunar SAR Competition Q&A] [answered]
Trying to see if we can hardcode in pathways for the tasks to guarantee we have enough battery to make it back to charging station. Would the locations of everything else (rovers, hab, etc outside of obstacles and the crash site) remain consistent too?

**Comment by DerryLin-Tamu** [comment, marked as answer]
Never mind I realized it was mentioned in the email.

**Comment by connorjak** [comment]
Right, only the obstacle and crash site locations vary between final task scenarios
